ignite/contrib/handlers/wandb_logger.py

```diff
@@ -226,6 +226,22 @@ def global_step_transform(*args, **kwargs):
                 global_step_transform=global_step_transform
             )
 
+        Another example where the State Attributes ``trainer.state.alpha`` and ``trainer.state.beta``
+        are also logged along with the NLL and Accuracy after each iteration:
+
+        .. code-block:: python
+
+            wandb_logger.attach(
+                trainer,
+                log_handler=OutputHandler(
+                    tag="training",
+                    metric_names=["nll", "accuracy"],
+                    state_attributes=["alpha", "beta"],
+                ),
+                event_name=Events.ITERATION_COMPLETED
+            )
```

Comment:
Actually, I think an example using `wandb_logger.attach_output_handler` would be shorter than `wandb_logger.attach`. Take a look at lines 221-226

Reply:
```suggestion
            wandb_logger.attach_output_handler(
                trainer,
                event_name=Events.ITERATION_COMPLETED,
                tag="training",
                metrics=["nll", "accuracy"],
                state_attributes=["alpha", "beta"],
            )
```
Do mean like this? It does reduce the line of initialising the Output Handler

Reply:
Yes, that's correct